src/html/index.html

```diff
@@ -5,12 +5,12 @@
     <title>Lab15_Gulp</title>
     <link rel="stylesheet" href="css/style.css">
     <script type="text/javascript" src="js/homepage.js"></script>
-    <script type="text/javascript" src="js/hello.js"></script>
   </head>
   <body>
     <div class="greet">Lab for <span class="name" id="brian">Brian Sheen</span></div>
-    <div id="hello"></div>
-    <div id="timelineColumn"></div>
-    <div id="userTweetsColumn"></div>
+    <div id="twitterTabs"></div>
+    <div id="timelineColumn" class="tabContent"></div>
```

Comment:
since you're using react already, you should build this in a react component as well